### PR TITLE
Add day view, reworked

### PR DESCRIPTION
**Pull request by Graywing13**
- Back button
- vertical day view
- add emoji into the mock content
- some minor css adjustments elsewhere

### CODE REVIEW

File: client/src/components/Views/DayViewContent.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  useEffect(() => {
+    // TODO edit this entire block to fetch from backend
+    const mockData = TripViewJson[activeDayNumber - 1]
+    setStages(mockData['stages'])
+  }, [activeDayNumber])
+
+  const generateStage = useCallback((stageInfo, index) => {
+    return (
+      <div className='grid grid-cols-5 py-2' key={`stage-${index}`}>
+        <div className='z-[5] w-20 text-center text-5xl'>
```
Comment (vee-16) [inline client/src/components/Views/DayViewContent.js:23]: We could use mx-auto later on to have the emoji responsive, so adjusting screen size won't hide location name and description

**Review by vee-16 — APPROVED** [review]
The rest looks works well on my end

File: client/src/components/Views/DayViewContent.js (hunk 1 above)
Comment (Graywing13) [reply client/src/components/Views/DayViewContent.js:23]: good idea looking into it

Comment (Graywing13) [reply client/src/components/Views/DayViewContent.js:23]: Couldnt get container working but added a screen-responsive breakpoint for now

image

**Pull request merged**